static/docs/src.html

```diff
@@ -384,6 +384,10 @@ <h2>Deprecated Options</h2>
       <h2>Common Errors</h2>
       <section data-include="Errors-FAQ" data-include-replace="true"></section>
     </section>
+    <section class="appendix">
```

Comment:
I shows up as "D.1 Getting Started" which is kinda noise in sidebar.

```suggestion
    <section class="appendix" data-max-toc="1">
```

Should also probably add `data-include-replace="true"` to `data-include`